galter_subjects_utils/updater.py

```diff
@@ -133,17 +157,26 @@ def update_rdm_record(record, ops_data, logger, keep_trace):
                 keep_trace.trace(record, op_data["subject"])
             logger.log(record.pid.pid_value, delta=op_data)
 
+    # Make sure subjects are deduplicated
+    record["metadata"]["subjects"] = deduplicate_subjects(
```

Comment:
Fix for issue found when qualifier to non qualifier transition ran. This uses the same approach as back in galter-ir-importer.